db/upgrade.php

```diff
@@ -30,10 +30,11 @@
  */
 function xmldb_tool_lcbackupcourselogstep_upgrade($oldversion) {
     global $DB;
+    $dbman = $DB->get_manager();
 
     if ($oldversion < 2024102000) {
 
-        $table = new xmldb_table('tool_lcbackupcourselogstep_metadata');
+        $table = new xmldb_table('tool_lcbackupcourselogstep_m');
```

Comment:
I'm a bit confused. Why do we need to change the table name in the past upgrade steps? Shouldn't we have `tool_lcbackupcourselogstep_metadata` table in that older version of the plugin?

Reply:
@gbarat87 as discussed internally, upgrade script 2024102000 can be remained as is, no need to replace table name in `if ($oldversion < 2024102000) {`